lib/galaxy/util/inflection.py

```diff
@@ -138,13 +138,22 @@ class English(Base):
 
     This is the default Inflector for the Inflector obj
     """
+    uncountable_words = ('equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep', 'sms')
```

Comment:
I think species, fish (unless meaning food) and sheep are countable nouns

Reply:
oh, I see, it's an existing inflection script. And these values refer to how the plural form should be constructed - in which case the logic is correct, I think. So, never mind. (still, poor choice of variable name by the script's original author, I think.)

Reply:
Yep, in https://github.com/bermi/Python-Inflector/pull/2/files (from which I took some ideas) for Spanish they are called `non_changing_words`. I can change it here too if you want.

Reply:
I don't mind leaving it as is. It's a very minor thing, it's good enough; besides touching the linguistic side of it might lead us down the garden path - I'm sure there are many more nouns that are pluralized according to the same rule.